### PR TITLE
Release/3.5.x: Issue #6342: Extended the docs of `text_enabled` param for plugins #6349

### PR DESCRIPTION
**Pull request by jeroenpeters1986**
### Summary
Added a little bit more information on `text_enabled` to help developers discover this powerful feature

### Links to related discussion
I reported this in issue #6342, which has been moved to https://github.com/divio/djangocms-text-ckeditor/issues/478

### Proposed changes in this pull request
Just some additions to the docs of djangocms-text-ckeditor

### Documentation checklist

* [x] I have updated CHANGELOG.txt if appropriate
* [x] I have updated the release notes document if appropriate, with:
    * [x] general notes
    * [x] bug-fixes
    * [x] improvements/new features
    * [x] backwards-incompatible changes
    * [x] required upgrade steps
    * [x] names of contributors
* [x] **I have updated other documentation**
* [x] **I have added my name to the AUTHORS file**
* [ ] This PR's documentation has been approved by Daniele Procida

### CODE REVIEW

File: docs/reference/plugins.rst [hunk 1]
```diff
@@ -191,13 +191,20 @@ CMSPluginBase Attributes and Methods Reference
...
+        be previewed in the text editor.
 
-        Not all plugins are usable in text plugins. If your plugin *is* usable in a text plugin:
+        Of course, not all plugins are usable in text plugins. Therefore the default of this
+        attribute is ``False``. If your plugin *is* usable in a text plugin:
 
         * set this to ``True``
-        * make sure your plugin provides its own :meth:`icon_src`
+        * make sure your plugin provides its own :meth"`icon_alt`, this will be used as a tooltip in 
+        the text-editor and comes in handy when you use multiple plugins in your text.
```
Comment (PeterW-LWL) [inline docs/reference/plugins.rst]: @jeroenpeters1986 I think you are missing 2 spaces in front. Maybe that it will be merged.

**Comment by jeroenpeters1986** [comment]
Thanks for noticing @PeterW-LWL, I have updated it

**Comment by jeroenpeters1986** [comment]
Can Travis CI be restarted? I cant seem to do that, I don't think this PR broke it but it was another error

**Comment by jeroenpeters1986** [comment]
All checks have been passed, is it possible to complete this PR?

**Comment by jeroenpeters1986** [comment]
Hi @evildmp, there is one to-do item on this PR with your name on it. Could you please review the text I added so this PR can be merged and closed?

**Comment by evildmp** [comment]
[Merged into 3.6](https://github.com/divio/django-cms/commit/a6b56a884e6ef2d51114b94a14241d75fdf3a898). Bedankt @jeroenpeters1986.

**Comment by jeroenpeters1986** [comment]
Thanks you are very welcome!

**Pull request closed without merging**